docs/pages/faq.mdx

```diff
@@ -174,11 +174,9 @@ For more information on what current limitations exist with EAS, see the followi
   Icon={Bell03Icon}
 />
 
-#### Bare workflow
+#### EAS & bare React Native Projects
```

Comment:
```suggestion
#### EAS and bare React Native projects
```

As per our writing style guide, we avoid using symbols and instead prefer to spell them out. Let's update the section heading here to maintain that consistency.

Also, in heading sections, we prefer to use sentence case instead of title case.